packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -45,17 +45,18 @@ const AppContents = observer(({ children }) => {
 
     React.useEffect(() => {
         if (is_logged_in && user_id) {
-            RudderStack.setAccountType(loginid.substring(0, 2));
-            RudderStack.identifyEvent(user_id, {
-                language: getLanguage().toLowerCase() || 'en',
+            Analytics.setAttributes({
+                account_type: loginid.substring(0, 2),
+                user_id,
+                user_language: getLanguage().toLowerCase() || 'en',
```

Comment:
I'd like to set these params once in `packages/core/src/Stores/client-store.js`

Reply:
I think this is fine and also in the other places as well, the values are getting changed so it should be updated for our rudderstack instance